plugins/extractors/bigquery/bigquery.go

```diff
@@ -164,22 +166,28 @@ func (e *Extractor) Init(ctx context.Context, config plugins.Config) error {
 
 // Extract checks if the table is valid and extracts the table schema
 func (e *Extractor) Extract(ctx context.Context, emit plugins.Emit) error {
+	pageSize := pickFirstNonZero(e.config.DatasetPageSize, e.config.MaxPageSize, 1)
```

Comment:
Lets use a larger default page size. Could be 10 for datasets and 50 for tables.